src/plugins/dll/src/SharedLibrary.cpp

```diff
@@ -38,54 +38,54 @@ bool SharedLibrary::open(const char *filename) {
     close();
 #if defined(WIN32)
     implementation = (void*)LoadLibrary(filename);
-    LPTSTR msg = NULL;
+    LPTSTR msg = nullptr;
     FormatMessage(
        FORMAT_MESSAGE_FROM_SYSTEM |FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_IGNORE_INSERTS,
-       NULL, GetLastError(), MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
-       (LPTSTR)&msg, 0, NULL);
+       nullptr, GetLastError(), MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+       (LPTSTR)&msg, 0, nullptr);
 
-    if(msg != NULL) {
+    if(msg != nullptr) {
```

Comment:
I would change this checks in `if(msg)` and the negative check `if(msg == nullptr)` in `if(!msg)`.

Reply:
I think that it is better to move the change you requested in a separated PR, we have to do some cleanups (e.g invert `RTF_ASSERT_IF` logic) and we can add also these changes for the nullptr checks.

Reply:
Ok!